go/libkb/identity_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import (
+	"testing"
+)
+
+func TestParseIdentity(t *testing.T) {
+	for i, identLine := range inputIdentLines {
+		expected := expectedIdentities[i]
+		identity, err := ParseIdentity(identLine)
+		if err != nil {
+			t.Errorf("error parsing identity: %s", err)
+		} else {
+			if *identity != expected {
+				t.Errorf("identity differs from expected\n%s\n%s", identity, expected)
+			}
+		}
+	}
+}
```

Comment:
Might it help to use anonymous structs?  https://talks.golang.org/2012/10things.slide#1 I think that might clean things up a bit here (and they're good to know about)